tests/cpp/test_sdpa_node.cpp

```diff
@@ -826,6 +830,137 @@ TEST_F(SDPATest, Sharded_SdpaFwd) {
   validateSdpaFwdOutputs(nvf_out, aten_out);
 }
 
+// TODO: Remove/update when https://github.com/NVIDIA/Fuser/issues/2563 is
+// resolved.
+TEST_F(SDPATest, Sharded_SdpaBwd) {
+  NVFUSER_TEST_CUDA_ARCH_GUARD(8, 0);
+  at::manual_seed(0);
```

Comment:
Is this for dropout? I don't see the random seed gets reset in this test, so I'm confused about its purpose.

Reply:
Ooops not needed, was doing too much copy paste